plugins/palo_alto_pan_os/setup.py

```diff
@@ -2,10 +2,10 @@
 from setuptools import setup, find_packages
 
 
-setup(name="palo_alto_pan_os-rapid7-plugin",
-      version="6.1.3",
+setup(name="palo_alto_pan_os-bindu-plugin",
```

Comment:
```suggestion
setup(name="palo_alto_pan_os-rapid7-plugin",
```

Reply:
Fixed in latest commit. Regenerated the checksum and other files